src/components/BuildBaronAndAnnotations/AnnotationNote.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from "react";
+import { useMutation } from "@apollo/client";
+import styled from "@emotion/styled";
+import Button, { Variant, Size } from "@leafygreen-ui/button";
+import { Disclaimer } from "@leafygreen-ui/typography";
+import { Input, Tooltip } from "antd";
+import { ConditionalWrapper } from "components/ConditionalWrapper";
+import { useBannerDispatchContext } from "context/banners";
+import {
+  EditAnnotationNoteMutation,
+  EditAnnotationNoteMutationVariables,
+  Note,
+} from "gql/generated/types";
+import { EDIT_ANNOTATION_NOTE } from "gql/mutations";
+import { getDateCopy } from "utils/string";
+import {
+  TicketsTitle,
+  TitleAndButtons,
+  MetaDataWrapper,
+  ButtonWrapper,
+} from "./BBComponents";
+
+const { TextArea } = Input;
+
+interface Props {
+  note: Note;
+  taskId: string;
+  execution: number;
+  userCanModify: boolean;
+}
+
+export const AnnotationNote: React.FC<Props> = ({
+  note,
+  taskId,
+  execution,
+  userCanModify,
+}) => {
+  const originalMessage = note?.message;
+  const dispatchBanner = useBannerDispatchContext();
+  const [newMessage, setMessage] = useState(originalMessage);
+  const [
+    updateAnnotationNote,
+    { loading: loadingAnnotationNote },
+  ] = useMutation<
+    EditAnnotationNoteMutation,
+    EditAnnotationNoteMutationVariables
+  >(EDIT_ANNOTATION_NOTE, {
+    onCompleted: () => {
+      dispatchBanner.successBanner(`Annotation note updated.`);
+    },
+    onError(error) {
+      dispatchBanner.errorBanner(
+        `There was an error updating this note: ${error.message}`
+      );
+    },
+    refetchQueries: ["GetTask"],
+  });
+  const saveAnnotationNote = () => {
+    updateAnnotationNote({
+      variables: {
+        taskId,
+        execution,
+        originalMessage,
+        newMessage,
+      },
+    });
+  };
+
+  return (
+    <TitleAndButtons>
+      <TicketsTitle>Note</TicketsTitle>
+      {note && (
+        <MetaDataWrapper data-cy={`${originalMessage}-metadata`}>
+          <Disclaimer>
+            Updated: {getDateCopy(note.source.time, null, true)}{" "}
+          </Disclaimer>
+          <Disclaimer>Last Edited By: {note.source.author}</Disclaimer>
+        </MetaDataWrapper>
+      )}
+      <StyledTextArea
+        id="noteInput"
+        rows={2}
+        value={newMessage}
+        onChange={(e) => setMessage(e.target.value)}
+      />
```

Comment:
Nit: would it be possible to add more space between the metadata (author, time...) and the note, as well as between the note and the button? If you click on the inspect button on the design, it lets you click on individual items and view how much space there is between items (in pixels).